<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAAligmentEditWidget</class>
 <widget class="QWidget" name="DAAligmentEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>125</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aligment Edit Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButtonAligmentLeft">
     <property name="text">
      <string>Aligment Left</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/AligmentLeft.svg</normaloff>:/DACommonWidgets/icon/AligmentLeft.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAligmentRight">
     <property name="text">
      <string>Aligment Right</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/AligmentRight.svg</normaloff>:/DACommonWidgets/icon/AligmentRight.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAligmentTop">
     <property name="text">
      <string>Aligment Top</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/AligmentTop.svg</normaloff>:/DACommonWidgets/icon/AligmentTop.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAligmentBottom">
     <property name="text">
      <string>Aligment Bottom</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/AligmentBottom.svg</normaloff>:/DACommonWidgets/icon/AligmentBottom.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAligmentCenter">
     <property name="text">
      <string>Aligment Center</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/DACommonWidgets/icon/AligmentCenter.svg</normaloff>:/DACommonWidgets/icon/AligmentCenter.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
